src/app/app.component.ts

```diff
@@ -1,18 +1,25 @@
 import { Component } from '@angular/core';
 import { TasksComponent } from './tasks/tasks.component';
 import { NavComponent } from './shared/nav/nav.component';
+import { TranslateModule, TranslateService } from '@ngx-translate/core';
 
 @Component({
   selector: 'app-root',
   standalone: true,
   imports: [
     TasksComponent,
-    NavComponent
+    NavComponent,
+    TranslateModule
   ],
   templateUrl: './app.component.html',
   styleUrl: './app.component.scss'
 })
 export class AppComponent {
   title = 'task-manager';
 
+  constructor(private translate: TranslateService) {
+    this.translate.addLangs(['en', 'lv']);
```

Comment:
you could move all of the translation logic to a separate service maybe, to cleanup code and do all the initial setting etc on the service?